CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- New `minicart-layout`, `minicart-base-content`, `minicart-product-list` and `minicart-summary` interfaces.
```

Comment:
Avoid using -layout suffix for this

Reply:
Good catch! I updated the interface names but forgot to update them in the CHANGELOG!